crates/glaredb/src/local.rs

```diff
@@ -73,25 +74,23 @@ impl LocalSession {
                     format!("Connected to remote GlareDB server: {}", u.cyan()),
                 )
             } else {
-                let tls_conf = if opts.disable_tls {
-                    None
-                } else {
-                    match (opts.ca_cert_path.clone(), opts.domain.clone()) {
-                        (Some(ca_cert_path), Some(domain)) => Some(TlsConfig {
-                            ca_cert_path,
-                            domain,
-                        }),
-                        _ => {
-                            return Err(anyhow!(
-                                "Specify ca-cert-path and domain in --args for TLS"
-                            ));
-                        }
-                    }
-                };
-
                 let mut url: ProxyDestination = url.try_into()?;
-                url = url.with_tls(tls_conf);
-                let client = RemoteClient::connect_with_proxy_destination(url).await?;
+
+                let mut ca_info = None;
+                if !opts.disable_tls {
+                    url.dst
+                        .set_scheme("https")
+                        .expect("not able to convert http to https");
+
+                    ca_info = Some(
+                        reqwest::get(format!("{}/api/internal/databases/authenticate", url.dst))
```

Comment:
also, is there a pooled client that we could use?

Reply:
I don't have a good answer here, might appreciate @vrongmeal 's take

Reply:
We don't need to worry about a pooled client for an HTTP request here. Seems like this was removed.